### DISCUSSION TITLE
Custom Menus

### DISCUSSION THREAD

**Discussion by MemTheBun** [DEMOLITIONDON96/Demolition-Engine#15] [Ideas (Softcode)]
example

a hx or lua or json that adds a menu or delete it

and use states

**Comment by RealDealJinx** [comment]
fuck off /j

**Reply by MemTheBun** [reply]
ayo why rude

**Comment by ErenKaya31** [comment]


**Reply by ThatOneSillyGuyDon** [reply]
Damn, someone’s being really fucking rude

**Reply by ghost** [reply]
(Fuck off go use yoshi engine)

**Comment by charlesisfeline** [comment]
good idea
but 
yoshi engine exists

**Reply by charlesisfeline** [reply]
and like, learn source then

**Comment by ThatOneSillyGuyDon** [comment]
Softcoded Menus are still being worked on, in the meantime, as @SonicProYT said, learn how to use source, or just use Yoshi Engine at the time being

**Reply by MemTheBun** [reply]
yea Cat Mayhem use yoshi for now
